packages/yoga/src/Snackbar/web/Snackbar.jsx

```diff
@@ -0,0 +1,190 @@
+import React, { useEffect } from 'react';
+import styled, { withTheme } from 'styled-components';
+import { func, string, bool, number, oneOf } from 'prop-types';
+
+import { media } from '@gympass/yoga-helpers';
+
+import { Close } from '@gympass/yoga-icons';
+
+import { theme } from '../../Theme';
+
+import Box from '../../Box';
+import Button from '../../Button';
+import Icon from '../../Icon';
+import Text from '../../Text';
+
+const IconWrapper = styled.div`
+  display: flex;
+  align-items: center;
+
+  margin-right: ${theme.spacing.xxsmall}px;
+`;
+
+const IconButtonWrapper = styled.div`
+  display: flex;
+  align-items: center;
+
+  cursor: pointer;
+
+  &:hover {
+    svg {
+      fill: ${theme.colors.text.secondary};
+    }
+  }
+`;
+
+const ActionsWrapper = styled.aside`
+  display: flex;
+  align-items: center;
+
+  svg {
+    margin-left: ${theme.spacing.small}px;
+    margin-right: ${theme.spacing.zero};
+  }
+`;
+
+const StyledSnackbar = styled.div`
+  ${({
+    variant,
+    theme: {
+      yoga: {
+        components: { snackbar },
+      },
+    },
+  }) => `
+      display: flex;
+      align-items: center;
+      justify-content: space-between;
+  
+      position: absolute;
+      bottom: ${snackbar.position.mobile.bottom}px;
+      right: ${snackbar.position.mobile.right}px;
+      left: ${snackbar.position.mobile.left}px;
+  
+      min-width: ${snackbar.minWidth.mobile}px;
+      max-width: ${snackbar.maxWidth.default}px;
+  
+      min-height: ${snackbar.height.min}px;
+      max-height: ${snackbar.height.max}px;
+  
+      padding: ${snackbar.padding.default}px;
+      
+      border-radius: ${snackbar.border.radius}px;
+  
+      box-shadow: ${snackbar.shadow.default};
+
+      background-color: ${snackbar.variant.color[variant]};
+
+      z-index: 100;
```

Comment:
How can I add some fadein animation using `styled-components`?

Reply:
I tried, but I got an error message about using `css`  and I still didn't get how it works. Could you help me?

Reply:
@coelhucas Now i got! I used this [article](https://www.mariokandut.com/how-to-add-css-animations-with-styled-components/) as a guide

Reply:
Sorry, I didn't noticed this earlier. But glad you managed to do it! :)